libraries/core/src/main/java/com/paritytrading/philadelphia/FIXConnection.java

```diff
@@ -612,6 +612,15 @@ private void sendTestRequest(FIXValue testReqId) throws IOException {
         send(adminMessage);
     }
 
+    public void sendResendRequest(long beginSeqNo) throws IOException {
+        prepare(adminMessage, ResendRequest);
+
+        adminMessage.addField(BeginSeqNo).setInt(beginSeqNo);
+        adminMessage.addField(EndSeqNo).setInt(0);
+
+        send(adminMessage);
+    }
```

Comment:
Let's move this after the `sendLogon` method and add an appropriate Javadoc comment. For example:
```java
/**
 * Send a ResendRequest(2) message.
 *
 * @param beginSeqNo the BeginSeqNo(7)
 * @throws IOException if an I/O error occurs
 */
 public void sendResendRequest(long beginSeqNo) throws IOException {
     ...
```